src/honeysql/helpers.clj

```diff
@@ -93,6 +93,12 @@
 (defhelper merge-right-join [m clauses]
   (update-in m [:right-join] concat clauses))
 
+(defhelper full-join [m clauses]
+  (assoc m :full-join clauses))
+
+(defhelper merge-full-join [m clauses]
+  (assoc m :merge-full-join clauses))
```

Comment:
@justindell Do you actually want to assoc on a :merge-full-join key or do you want to merge with the existing :full-join key (as in the merge-right-join helper above)